review-tools/addrev

```diff
@@ -14,7 +14,10 @@ my $useself = 1;
 my $my_email;
 
 foreach (@ARGV) {
-    if (/^\@.+$/) {
+    if (/^(--prnum=)?(\d{1,6}+)$/) {
```

Comment:
I understand that you want to support all numeric 7 character git commit id. Hopefully the PR numbers won't be in millions range too soon. :grin:

Reply:
Yes. Recently the PR numbers became 5-digit, and it will certainly take a while until 6 digits are needed
By that time likely also more than 7 hex digits will be needed to identify commits,
so then our (grand)children can simultaneously increment both limits by one :sweat_smile: